src/pybind/mgr/dashboard/tox.ini

```diff
@@ -62,22 +66,18 @@ addopts =
     --rcfile=.pylintrc
     --jobs={[pylint]jobs}
 
-[rstlint]
-dirs =
-    README.rst
-    HACKING.rst
-
 [testenv:lint]
 basepython=python3
 deps =
     -rrequirements.txt
     -cconstraints.txt
     -rrequirements-lint.txt
+whitelist_externals = *
 commands =
-    lint: flake8 {posargs}
+    lint: flake8 --config=tox.ini {posargs}
     lint: pylint {[pylint]addopts} {posargs:{[pylint]dirs}}
-    lint: rstcheck --report info --debug {posargs:{[rstlint]dirs}}
-
+    lint: bash -ec 'if [[ -z "{posargs}" ]]; then \
+            rstcheck --report info --debug $(find . -maxdepth 1 -iname "*.rst" | tr "\n" " "); fi'
```

Comment:
- We should split `lint` into 3 subrecipes: `pylint`, `flake8` and `rstcheck`, so it's easy to feed specific files into this.
- Optionally: while it doesn't make much change for this case (as RST files are not added everyday), I think it's better to allow new files to be added in a variable, as it allows a more controlled checking.
- Optionally: Why not something like filtering out non-rst files from {posargs}?

Reply:
@epuertat I kept ```[testenv:lint]````without {posargs} (for CI environments) and added the aforementioned subrecipes that accept {posargs}.